### PR TITLE
[tlgen/docs] Expand slightly to explain what node names should be

### PR DESCRIPTION
**Pull request by rswarbrick**
The first patch gets rid of some debugging lines that were appearing in the output. The second expands the tlgen documentation a little bit to explain exactly how a user should be naming devices.

### CODE REVIEW

**Comment by rswarbrick** [comment]
@tjaychen, @eunchan: Ping! I think this is pretty trivial, but it would be nice to get merged.

File: doc/rm/crossbar_tool/index.md [hunk 1]
```diff
@@ -30,12 +30,26 @@ While generating the RTL, the tool adds detailed connection information in the f
 ## Configuration File Format
 
 The `tlgen` script reads an Hjson file containing the crossbar connections and the host and device information.
-It describes a generic Directed Acyclic Graph (DAG) with some additional clock information and steering information.
+The two main sections of the configuration file are the list of nodes and the list of connections.
+Together, these describe a a generic Directed Acyclic Graph (DAG) with some additional clock information and steering information.
```
Comment (msfschaffner) [inline doc/rm/crossbar_tool/index.md:34]: nit: repeated "a"

**Comment by tjaychen** [comment]
o yeah this lgtm, but i thought @eunchan was the more appropriate person to give a thumbs up.

**Review by msfschaffner — APPROVED** [review]

**Review by eunchan — APPROVED** [review]
LGTM. Thanks for clarifying how to write Hjson configuration for Xbar.

File: doc/rm/crossbar_tool/index.md (hunk 1 above)
Comment (rswarbrick) [reply doc/rm/crossbar_tool/index.md]: Thanks!

**Pull request merged**